test/erlcloud_aws_tests.erl

```diff
@@ -360,6 +360,10 @@ profiles_assume_cleanup(P) ->
     profiles_test_cleanup(P),
     meck:unload( erlcloud_sts ).
 
+default_config_region_sunny_test() ->
+    Region = <<"ca-central-1">>,
```

Comment:
should not this be us-east-1? why ca-central-1 ?

Reply:
here may be just `<<"some region">>`. that does not matter for this test.